### DISCUSSION TITLE
Upgrade to Highcharts

### DISCUSSION THREAD

**Discussion by Kamsou** [richardeschloss/nuxt-highcharts#78] [Q&A]
Hi @richardeschloss ! 

Thanks for this lib, as it helped me a lot. 
However, I have a problem with XRange. And one of the highcharts contributors said it was solved on version 11.4.1
Would there be any way to mount the highchart version of the module?  or does it sound complicated?

**Comment by richardeschloss** [comment]
`npm i` should install the latest v11 highcharts (11.4.8) which would include that.. Can you please confirm the highcharts version by searching package-lock.json file for "highcharts" (with the double quotes)?
